### ISSUE TITLE
Players randomly receive ammo

### ISSUE THREAD

**Issue by samm81** [alanhdu/hunt#39]
This might happen when a wall is put into the arena, and the game thinks it's a player?

**Comment by alanhdu** [comment]
Hm...I can't seem to reproduce. 

I know that you regain 5 ammo every time another player enters the arena (either from dying or from logging in the first time). Maybe that's what's happening?

**Comment by samm81** [comment]
Yeah, I can't either unfortunately. It was after 10 or 20 minutes of playing, with 3 people. But while trying to replicate it I found that I oftentimes couldn't get two people in the same game...
